mettle/extensions/lua/lua.c

```diff
@@ -0,0 +1,85 @@
+#include "extension.h"
+
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
+
+#include "lua.h"
+
+/*
+ * Override 'print' so the extension won't crash
+ * Thanks @timwr for the link: https://gist.github.com/5at/3671566
+ */
+static int print_to_framework_side(lua_State *lua) {
+	int nargs = lua_gettop(lua);
+	for (int i = 0; i < nargs; i++) {
+		const char *str = lua_tostring(lua, i);
+
+		// Send as TLV
+		// Need to figure out a way to this
```

Comment:
Why not just concatenate everything into a buffer and add the buffer to a tlv at the bottom of request_execute_code?